modules/identity-provider-creating-htpasswd-file-windows.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-htpasswd-identity-provider.adoc
+
+[id="identity-provider-creating-htpasswd-file-windows-{context}"]
+= Creating an HTPasswd file using Windows
+
+To use the HTPasswd identity provider, you must generate a flat file that
+contains the user names and passwords for your cluster by using
+link:http://httpd.apache.org/docs/2.4/programs/htpasswd.html[`htpasswd`].
+
+.Prerequisites
+
+* Have access to `htpasswd.exe`. This file is included in the `\bin`
+directory of many Apache httpd distributions.
+
+.Procedure
+
+. Create or update your with a user name and hashed password:
```

Comment:
missing a word in "Create or update your with". Looks like it was copied from the Linux file, which needs to be fixed as well.

Reply:
I changed this to be `flat file`, since that's how it is introduced in the original paragraph, for both files.